src/sentry/api/base.py

```diff
@@ -213,14 +218,28 @@ def permission_denied(self, request, message=None, code=None):
         the appropriate exception according to parent DRF function.
         """
         permissions = self.get_permissions()
+        if request.user.is_authenticated and len(permissions) == 1:
+            permission_cls = permissions[0]
+            use_staff = features.has("auth:enterprise-staff-cookie", actor=request.user)
```

Comment:
```suggestion
            enforce_staff_permission  = features.has("auth:enterprise-staff-cookie", actor=request.user)
```
nit: naming could be more specific?